index.d.ts

```diff
@@ -503,7 +494,7 @@ declare namespace Moleculer {
 		validate(params: GenericObject, schema: GenericObject): boolean;
 	}
 
-	class LoggerHelper {
+	class Logger {
```

Comment:
sorry, i am not using typescipt currently. i just use es in node8  and wrote in vs code, editor can find index.d.ts and make some suggestion for me , when i wrote  const { Logger } = require('moleculer'), current type define will  let me   
get this result  conts { LoggerHelper } = require('moleculer'), but in index.js there is no LoggerHelper class to export, if we need Logger interface  for other logger, how about change index.js to real export LoggerHelper class?

Reply:
because right now moleculer  do not log for any request ,in my logger ,i wish any request has is requestID and auto log ,so i wrote middleware for it, and so i use Logger class directly .
and other people who has same wish,like #238 , how do you think this issue. i didnt use this framework in production , i still researching for it, i think it is a great microservice framework for nodejs and want to help and make it better. this is my first pull request in github. so i wish i can  fix this little puzzle problem for it

Reply:
I'll look more into the code and come back with a possible solution

Reply:
I'll look more into the code and come back with a possible solution

Reply:
Keep the original interface with a new name, something like `BaseLogger` 
Change `index.d.ts` line 168 (BrokerOptions.logger) to the new interface name and line 564 (Transporters.Base.logger) to `LoggerInstance`. Also `LoggerInstance` should implement the `BaseLogger` interface. 

`Logger` would also not be a `class` as you can't do `new Logger()` to get an instance. Define it as
```
export const Logger = {
    ...
};
```

Reply:
Keep the original interface with a new name, something like `BaseLogger` 
Change `index.d.ts` line 168 (BrokerOptions.logger) to the new interface name and line 564 (Transporters.Base.logger) to `LoggerInstance`. Also `LoggerInstance` should implement the `BaseLogger` interface. 

`Logger` would also not be a `class` as you can't do `new Logger()` to get an instance. Define it as
```
export const Logger = {
    ...
};
```